public/stylesheets/site/2.0/26-media-narrow.css

```diff
@@ -82,21 +82,22 @@ body .narrow-shown {
 
 /* 05 region main */
 
-#main {
-  position: static;
-}
-
 #main, #main.dashboard {
-  margin: auto;
+  position: static;
 }
 
-#main.dashboard .index, form.filters, .works-index .index, .collections-index .index, .bookmarks-index .index, #inner form dd, #inner form dt {
```

Comment:
lolsob, it was fun trying to untangle this pile of selectors to figure out what was causing the issue in [my comment below](https://github.com/otwcode/otwarchive/pull/2363/files#r52824821). I don't want to do that ever again, so:

We forgot to simplify `.works-index .index, .collections-index .index, .bookmarks-index .index` to `.filtered .index` in https://github.com/otwcode/otwarchive/commit/ae82d93053ed9f9c68eba239ec66095c483e0247, whoops. 

`#main.dashboard .index` only needed the `float` style -- the width styling was partly to blame for the bug -- and `#main` was an unnecessary part of the selector (the style it's overriding comes from `.dashboard .index`). Likewise, including `#inner` for the form elements was overkill for fixing width issues that could've been resolved by including `form .meta dd, form .meta dt`.